getFunctions.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * @param array $shape The shape data for the required image.
+ *
+ * @return string The Img tag containing the shape and alternate text.
+ */
+
+function getNameImg(array $shape): string {
+    return "<img alt=\"" . $shape["ShapeName"] . "\"src=\"" . $shape["imgurl"] ."\" />";
```

Comment:
when you need to use quotes inside a string, wrap the string in the opposite kind of quotes so you dont have to escape it. so here, use single quotes to contain the string. keeps it cleaner